doc/src/learn/sui-compared.md

```diff
@@ -123,6 +123,20 @@ Sui mitigates this with checkpoints. A checkpoint is established every time an i
 
 Sui uses the state commitment that arrives upon epoch change. Sui requires a single answer from the multiple validators and leverages an accessory protocol to derive the hash representing the state of the blockchain. This protocol consumes little bandwidth and does not impede the ingestion of transactions. Validators produce checkpoints at every epoch change. Sui requires the validators to also produce checkpoints even more frequently. So users may use these checkpoints to audit the blockchain with some effort.
 
+## Summary
+
+|                 **Concept**                 |                  **Sui**                  |  **Traditional blockchain** |
+|:-------------------------------------------:|:-----------------------------------------:|:---------------------------:|
+| _Unit of storage_                           | object                                    | account                     |
+| _Transaction execution_                     | parallel (for independent transactions)   | sequential                  |
+| _Transactions flow_                         | proactively talk with validators          | fire-and-forget broadcast   |
+| _Transactions validation_                   | individually                              | batching them in the blocks |
+| _Locked data during transaction validation_ | only the relevant piece of data           | the whole chain             |
```

Comment:
do we want to more explicitly call out shared object and owned object because this is the inherent difference - shared object are sequenced through consensus, owned object can be highly parallelized.